{{cookiecutter.slug}}/compose-postgenerate.yml

```diff
@@ -0,0 +1,17 @@
+services:
+    backend:
+        build:
+            context: ./backend
+            dockerfile: Dockerfile-postgenerate
+        volumes:
+            - ./backend:/usr/src/app/backend
+    frontend:
+        build:
+            context: ./frontend.{{cookiecutter.frontend}}
+            dockerfile: Dockerfile-postgenerate
+        volumes:
+            - ./frontend:/usr/src/app/frontend
```

Comment:
Note that this step will copy the (empty) host frontend directory over the frontend directory in the container (which has now a `locale` directory). So the bind mount should be from `/frontend/src`.